docs/installation.rst

```diff
@@ -32,21 +32,24 @@ or, on windows,
 Fortran Compilers
 -----------------
 
-By default, sami2py uses gfortran and make to compile the fortran executables.  You can get make through pip
+By default, sami2py uses gfortran and make to compile the fortran executables.
+If you don't have a fortran compiler, gfortran is included as part of the latest
+gcc package.  You can get this from several locations.
+
+For Mac OS X, you can install gcc through package managers such as `brew <https://brew.sh/>`_.
+
+For windows, multiple setup options are discussed at https://www.scivision.dev/cmake-install-windows
```

Comment:
Do you want to leave a note about making sure the Windows tools are installed before the compiler is?

Reply:
Since I don't use windows, a suggested line would be good.

Reply:
I gave you my best suggested line above (and you know my equal incompetence with Windows).  Basically the problem you helped troubleshoot for AACGMV2 at ISEA.  If you don't want to bother, that's also fine.

Reply:
updated as best as i can